### ISSUE TITLE
Why does the released dataset not contain 81 million samples?

### ISSUE THREAD

**Issue by fispresent** [GabrielleTse/emind#7]
Hello, thank you for your work. When I accessed the dataset via the link you provided, I found that the training set only contains 496K samples and the validation set has just 4.47M samples. Could you please tell me where I can obtain the full dataset?
